tests/Microsoft.Identity.Test.Integration.net45/HeadlessTests/OboTests.cs

```diff
@@ -70,60 +70,6 @@ public async Task OBO_WithCache_MultipleUsers_Async()
             await RunOnBehalfOfTestAsync(aadUser2, true).ConfigureAwait(false);
         }
 
-        [TestMethod]
```

Comment:
Can you ignore them instead? We should fix timeouts.

Reply:
[Ignore] and add a github issue to fix them later